### ISSUE TITLE
Questions about the new update

### ISSUE THREAD

**Issue by cjdduarte** [Arthur-Milchior/anki-enhance-main-window#37]
After this new update, I noticed that this new version of the addon made the window cleaner.

New version (Today)

![image](https://user-images.githubusercontent.com/42243532/52469049-8b4eb200-2b71-11e9-9117-5864cae11eb1.png)

Old version
![image](https://user-images.githubusercontent.com/42243532/52469335-45461e00-2b72-11e9-9e8b-5d9b56f69ef2.png)

But looking at your github's change history, it seems like you've been renaming the names of some columns. Has this occurred?

Since I do not use all of them and I customize the columns a lot, I'm worried that an outdated version will occur in my version of META.JSON.

What has been fixed in this version other than the part that left the window cleaner?

**Comment by cjdduarte** [comment]
Alias, this "clean" version was purposeful? I did not find this change. That's right?

**Comment by Arthur-Milchior** [comment]
The 2nd way is what I was looking for all along. I don't know why 0 did
occur sometime, and sometime not.
May be I should make a configuration variable to choose whether to display
the 0s.

I added a new column. Normally, I renamed only a single column. And the
code edit your config file if it finds the old name. So normally, you won't
have trouble.
I try not to break anything. To be sure that valid configurations remain
valid.

Le ven. 8 févr. 2019 à 10:43, Carlos Duarte <notifications@github.com> a
écrit :
